docs/user-guide/data/search.md

```diff
@@ -0,0 +1,105 @@
+# Search and browse
+
+DSP-APP offers the possibility for the user to search in 3 different ways: **full-text** search, **advanced** search, and **expert** search (Gravsearch query). The search bar is always available in the header of each page, whether logged in or out.
+
+![Search bar with the 3 search modes: full-text, advanced, and expert](../../assets/images/search-3modes.png)
+
+## Full-text search
+
+The full-text search performs queries including one or more terms or phrases and returns data that matches search conditions. By default, the search is performed in all projects stored in DSP. However, it is possible to filter by the project using the menu "Filter by project" on the left side of the search bar.
```

Comment:
The full-text search performs queries including one or more terms or phrases and returns data that matches **the** search conditions. By default, the search is performed in all projects stored in DSP. However, it is possible to filter by ~~the~~ project using the ~~menu~~ "Filter by project" **menu** on the left side of the search bar.